compose.yml

```diff
@@ -1,35 +1,36 @@
 services:
   mongodb:
     image: mongo:6.0.13
-    command: mongod --port ${MONGO_PORT:-27017}
     networks:
       - cdp-tenant
     ports:
-      - "${MONGO_PORT:-27017}:${MONGO_PORT:-27017}"
+      - "${MONGO_PORT:-27017}:27017"
     volumes:
       - mongodb-data:/data
     restart: always
 
   gas:
     build: .
     ports:
-      - "${PORT:-3000}:${PORT:-3000}"
+      - "${GAS_PORT:-3000}:3000"
     links:
       - "mongodb:mongodb"
     depends_on:
       mongodb:
         condition: service_started
     environment:
-      PORT: ${PORT:-3000}
+      PORT: 3000
       NODE_ENV: development
-      MONGO_URI: mongodb://mongodb:${MONGO_PORT:-27017}/
+      MONGO_URI: mongodb://mongodb:27017/
     networks:
       - cdp-tenant
     develop:
       watch:
         - path: ./src
           target: /home/node/src
           action: sync+restart
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Allow the container to call endpoints on the host. Allows us to test POST actions.